avro/src/main/scala/hydra/avro/registry/SchemaRegistry.scala

```diff
@@ -111,13 +111,28 @@ object SchemaRegistry {
   private def getFromSchemaRegistryClient[F[_]: Sync](schemaRegistryClient: SchemaRegistryClient): SchemaRegistry[F] =
     new SchemaRegistry[F] {
 
+      private implicit class CheckKeySchemaEvolution(schemasF: F[List[Schema]]) {
+        def checkKeyEvolution(subject: String, newSchema: Schema): F[List[Schema]] = schemasF.flatTap[Unit] {
+          case _ if !subject.endsWith("-key") => Sync[F].unit
```

Comment:
Feels like a double negative. Can we do if subject.endsWith("-value") instead?

Reply:
I was mainly thinking that there _could_ be cases when we register subjects that don't end in `-key` or `-value`. But I guess without having a concrete use case for that you have a valid point. I will change it 🙂